website/public/js/filters/timezoneOffsetToUtc.js

```diff
@@ -0,0 +1,18 @@
+angular.module('habitrpg')
+  .filter('timezoneOffsetToUtc', function () {
+    return function (offset) {
+      if (offset > 0) {
+        var sign = '-';
+      } else {
+        var sign = '+';
+      }
+
+      offset = Math.abs(offset) / 60;
+
+      var hour = Math.floor(offset);
+
+      var minutes = (offset - hour) * 60;
```

Comment:
Could you please add something here so that if the number of minutes is less than 10, it prints a leading zero? For example, 10:00 instead of 10:0.